### ISSUE TITLE
Running "mathicsscript" gets FileNotFoundError: mathicsscript/data/mma-tables.json

### ISSUE THREAD

**Issue by tueda** [Mathics3/mathicsscript#37]
When I tried to start Mathicscript (3.0.0), which was installed in a clean virtual environment, it gave me
```
$ mathicsscript
Traceback (most recent call last):
  File "/home/tueda/tmp/test-mathicsscript/.venv/bin/mathicsscript", line 8, in <module>
    sys.exit(main())
  File "/home/tueda/tmp/test-mathicsscript/.venv/lib/python3.9/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/home/tueda/tmp/test-mathicsscript/.venv/lib/python3.9/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/home/tueda/tmp/test-mathicsscript/.venv/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/tueda/tmp/test-mathicsscript/.venv/lib/python3.9/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/home/tueda/tmp/test-mathicsscript/.venv/lib/python3.9/site-packages/mathicsscript/__main__.py", line 257, in main
    shell = TerminalShellPromptToolKit(
  File "/home/tueda/tmp/test-mathicsscript/.venv/lib/python3.9/site-packages/mathicsscript/termshell.py", line 368, in __init__
    self.completer = MathicsCompleter(self.definitions) if want_completion else None
  File "/home/tueda/tmp/test-mathicsscript/.venv/lib/python3.9/site-packages/mathicsscript/completion.py", line 75, in __init__
    with open(osp.join(get_datadir(), "mma-tables.json"), "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/tueda/tmp/test-mathicsscript/.venv/lib/python3.9/site-packages/mathicsscript/data/mma-tables.json'
```

The exception looks like something about the completion, so I tried the `--no-completion` option to switch off the compleiton, and it worked:
```
$ mathicsscript --no-completion

Mathicscript: 3.0.0, Mathics 2.2.0
on CPython 3.9.4 (default, Apr  7 2021, 05:45:11)
using SymPy 1.8, mpmath 1.2.1, numpy 1.20.3

Copyright (C) 2011-2021 The Mathics Team.
This program comes with ABSOLUTELY NO WARRANTY.
This is free software, and you are welcome to redistribute it
under certain conditions.
See the documentation for the full license.

Quit by evaluating Quit[] or by pressing CONTROL-D.

In[1]:= Quit[]


Goodbye!
```

**Comment by rocky** [comment]
Thanks for the report. 

It looks like I am missing the GNU Readline style inputrc files in the 3.0.0 distribution. 

Until I have a this fixed and automated better, have `Mathics_scanner` installed  and the program `admin-tools/make-tables.sh` from this (the `mathicsscript`) repository. 

I'll try to put out another release fixing this over the weekend.

**Comment by rocky** [comment]
Version [3.1.0 is now out on PyPI](https://pypi.org/project/mathicsscript/3.1.0/) and should address this problem while adding a few new features.

**Comment by tueda** [comment]
Thank you. I have confirmed that the issue has been resolved with 3.1.0.
